src/components/application_manager/test/application_impl_test.cc

```diff
@@ -565,7 +568,7 @@ TEST_F(ApplicationImplTest, SubscribeToSoftButton_UnsubscribeFromSoftButton) {
 
 TEST_F(ApplicationImplTest, ChangeSupportingAppHMIType_TypeNotNaviNotVoice) {
   smart_objects::SmartObject type_media;
-  type_media[0] = AppHMIType::MEDIA;
+  type_media[0] = mobile_apis::AppHMIType::MEDIA;
```

Comment:
@jacobkeeler 
- Reason to add: to avoid ambiguity with `AppHMIType` from policies and explicitly to specify that this one `AppHMIType` is used form mobile apis
- Reason not removed at the top of file: there are dozens of places of other entities usage from `mobile_apis` namespace.